docs/static/releasenotes.asciidoc

```diff
@@ -41,6 +42,44 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-10-0]]
+=== Logstash 8.10.0 Release Notes
+
+[[notable-8.10.0]]
+==== Notable issues fixed
+
+* Fixed bugs in DLQ 
+** throwing `java.nio.file.NoSuchFileException` when finalize the segment https://github.com/elastic/logstash/pull/15233[15233]
+** sorting DLQ file in wrong order https://github.com/elastic/logstash/pull/15246[#15246]
+
+[[dependencies-8.10.0]]
+==== Updates to dependencies
+
+* Updated JRuby to 9.4.2.0 https://github.com/elastic/logstash/pull/15283[#15283]
+* Removed custom bundler and used JRuby bundled bundler https://github.com/elastic/logstash/pull/15066[#15066]
+
+==== Plugins
+
+*Elasticsearch Output - 11.16.0*
+
+* Added support to Serverless Elasticsearch https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/1145[#114]
+
+*Elastic Serverless Forwarder 0.1.2*
```

Comment:
```suggestion
*Elastic Serverless Forwarder input 0.1.2*
```

Reply:
I'm not sure why the proper name and plugin type (input)  aren't being generated for this one.

Reply:
I did something wrong here